better-font-awesome.php

```diff
@@ -321,14 +325,20 @@ public function create_admin_page() {
         <div class="wrap">
             <?php screen_icon(); ?>
             <h2><?php echo $this->plugin_display_name; ?></h2>
-            <form method="post" action="options.php">
+            <form method="post" action="options.php" id="bfa-settings-form">
             <?php
                 // This prints out all hidden setting fields
                 settings_fields( 'option_group' );
                 do_settings_sections( self::SLUG );
-                submit_button();
-                echo $this->get_usage_text();
             ?>
+                <p>
+                    <span id="bfa-save-settings-button" class="button-primary"><?php _e( 'Save Settings', 'better-font-awesome' ); ?></span>
```

Comment:
@BraadMartin you probably want to use escape functions for security. So here for example instead of `_e( 'Save Settings', 'better-font-awesome' );` it's better to do `esc_html_e( 'Save Settings', 'better-font-awesome' );` so the code is translated, escaped and echoed.

Anything that's output should be escaped for better security: attributes, text.. Here is more here http://codex.wordpress.org/Validating_Sanitizing_and_Escaping_User_Data#Escaping:_Securing_Output :)